core/frontend/src/types/kraken.ts

```diff
@@ -0,0 +1,26 @@
+import { Dictionary } from '@/types/common'
+
+export interface Author {
+    name: string
+    email: string
+    about?: string
+}
+
+export interface Version {
+    permissions: any
+    requirements: string | null
+    tag: string
+}
+
+export interface ExtensionData {
+    name: string
+    description: string
+    docker: string
+    website: string
+    versions: Dictionary<Version>
+    authors?: Author[]
```

Comment:
It seems quite redundant (and hard to maintain consistently) for all the author data to be provided in each extension's metadata. Perhaps there's some way the author(s) can be specified at a higher level and then referred to from the extension data, which is then retrieved when creating/refreshing the manifest?

That may need to be specified in an independent global folder or database or something (separate to the extensions), with a file/entry per author and the filename/index is the reference key (maybe `organisation-author-name` like `br-willian` for easy grouping).

Alternatively each extensions repo could have a single file of authors that each of its extensions can optionally include, but that still has some redundancies when authors of independent extensions work together on an extension.

I suppose there would then need to be a `development` author or something shown by default for unmanaged extensions.

Reply:
I think we'll get there eventually, I'm trying to keep things simple. but for the final store with bazaar, we'll have a proper database.